modules/uptime-check/vars.tf

```diff
@@ -1,22 +1,22 @@
-variable "monitoring_project_id" {
+variable monitoring_project_id {
   type        = string
   description = "Project ID to create monitoring resources in"
   default     = "hiiretail-monitoring-prod-6500"
 }
 
-variable "uptime_checks" {
+variable uptime_checks {
   type        = any
   description = "The list of uptime checks configurations"
 }
 
-variable "notification_channels" {
+variable notification_channels {
   type        = list(any)
   description = "List of notificaton channel IDs"
   default     = []
 }
 
 variable labels {
-  type        = map
+  type        = map(any)
```

Comment:
`map` is a shorthand for `map(any)`, so you don't need `any`, it will accept any type as long as every element is the same type. The same for the list above